### ISSUE TITLE
[a11y] /contact page accessibility improvements

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#15339]
# Axe-core® Accessibility Results for /en-US/contact/

Page URL: https://www-dev.allizom.org/en-US/contact/

## axe-core found 5 violations

| #   | Description | Axe rule ID | WCAG | Impact | Count |
| --- | --- | --- | --- | --- | --- |
| [1](#1) | ARIA role should be appropriate for the element | aria-allowed-role | Best practice | minor | 1   |
| [2](#2) | Heading levels should only increase by one | heading-order | Best practice | moderate | 1   |
| [3](#3) | `<li>` elements must be contained in a `<ul>` or `<ol>` | listitem | WCAG 2 Level A, WCAG 1.3.1 | serious | 3   |

### Failed

1. ARIA role should be appropriate for the element

[Learn more](https://dequeuniversity.com/rules/axe/4.10/aria-allowed-role?application=playwright)

aria-allowed-role

Best practice

Ensure role attribute has an appropriate value for the element

minor

Issue Tags: cat.aria best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.category-tabs<br>```<br><br>**Element source**<br><br>```<br><ul class="category-tabs" role="navigation"><br>``` | Fix any of the following:<br><br>*   ARIA role navigation is not allowed for given element |

2. Heading levels should only increase by one

[Learn more](https://dequeuniversity.com/rules/axe/4.10/heading-order?application=playwright)

heading-order

Best practice

Ensure the order of headings is semantically correct

moderate

Issue Tags: cat.semantics best-practice

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.contact-section:nth-child(1) > h4<br>```<br><br>**Element source**<br><br>```<br><h4>Find us online</h4><br>``` | Fix any of the following:<br><br>*   Heading order invalid |

3. <li> elements must be contained in a <ul> or <ol>

[Learn more](https://dequeuniversity.com/rules/axe/4.10/listitem?application=playwright)

listitem

WCAG 2 Level A, WCAG 1.3.1

Ensure <li> elements are used semantically

serious

Issue Tags: cat.structure wcag2a wcag131 EN-301-549 EN-9.1.3.1

| #   | Issue Description | To solve this violation, you need to... |
| --- | --- | --- |
| 1   | **Element location**<br><br>```<br>.current<br>```<br><br>**Element source**<br><br>```<br><li class="current" data-id="contact"><br>    <a href="/en-US/contact/">Contact us</a><br>  </li><br>``` | Fix any of the following:<br><br>*   List item parent element has a role that is not role="list" |
| 2   | **Element location**<br><br>```<br>li[data-id="spaces"]<br>```<br><br>**Element source**<br><br>```<br><li data-id="spaces"><br>    <a href="/en-US/contact/spaces/">Spaces</a><br>  </li><br>``` | Fix any of the following:<br><br>*   List item parent element has a role that is not role="list" |
| 3   | **Element location**<br><br>```<br>li[data-id="communities"]<br>```<br><br>**Element source**<br><br>```<br><li data-id="communities"><br>    <a href="https://community.mozilla.org/groups/">Communities</a><br>  </li><br>``` | Fix any of the following:<br><br>*   List item parent element has a role that is not role="list" |
